configmodel/__init__.py

```diff
@@ -297,3 +298,43 @@ def deep_dict_merge(dest, merge, preserve=False):
         elif not preserve:
             dest[key] = copy.deepcopy(val)
     return dest
+
+def deep_traverse_and_replace_with_var_content(d: dict) -> dict:  # pylint: disable=C0103
+    """
+    Recursively traverse a dictionary and replace key-value pairs where the key ends with `_from_var`
+    The value is replaced with the corresponding environment variable value if available.
+    Existing truthy values for the base key will not be overwritten.
+    """
+    suffix = "_from_var"
+    keys_to_replace = [key for key in d if isinstance(key, str) and key.endswith(suffix)]
+
+    for key in keys_to_replace:
+        new_key = key[: -len(suffix)]
+
+        try:
```

Comment:
This function was simply moved from one package to another. The only change made is in this block, the replacement occurs only if the key is not already set with a value in the configuration. This behavior was introduced to match the original my_conf approach, which follows the same logic